### DISCUSSION TITLE
请问 DualSubs后续可以支持tubi吗

### DISCUSSION THREAD

**Discussion by 2583538** [NSRingo/.github#8] [Q&A 问答] [answered]
请问 DualSubs后续可以支持tubi吗，目前测试不可用。（youtube，pluto tv都可以），但是这个不可用，所以不知道是否是软件不支持？tubi有很多好看的电影。。。

**Comment by VirgilClyne** [comment, marked as answer]
这里是 `iRingo` 项目
`DualSubs` 项目请至
https://github.com/orgs/DualSubs/discussions
问询
